### DISCUSSION TITLE
请问支持 Rewrite吗

### DISCUSSION THREAD

**Discussion by JzwOnly** [eycorsican/leaf#145] [General]
拦截特点请求，修改请求参数和返回结构

**Comment by eycorsican** [comment]
不支持，有必要吗？

**Comment by EkkoG** [comment]
您的使用场景是什么呢？个人来说，目标好像就 google.cn 跳转到 google.com 用到了这个特性，还是因为苹果内置了 google.cn 不改

**Comment by JzwOnly** [comment]
@ciepy 主要是重定向问题， 修改response只是针对个别APP开屏广告 用`reject` 仍然会有3s
